src/lib/mdns/ServiceNaming.cpp

```diff
@@ -119,5 +119,38 @@ CHIP_ERROR MakeHostName(char * buffer, size_t bufferLen, const chip::ByteSpan &
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR MakeServiceSubtype(char * buffer, size_t bufferLen, DiscoveryFilter subtype)
+{
+    size_t requiredSize;
+    switch (subtype.type)
+    {
+    case DiscoveryFilterType::kShort:
+        requiredSize = snprintf(buffer, bufferLen, "_S%03u", subtype.code);
```

Comment:
Yes. I added a note in the commit description, but it doesn't show in the main PR for some reason. I realize these have changed, but I'd prefer to make that change in a separate PR because that change will break discovery between a controller and a device that are using different versions of the SDK. I think that change needs to go in alone, and with a very widespread PSA.

Reply:
Ok, sounds good to me.